### ISSUE TITLE
Active events are not showing for an Admin before filtering 

### ISSUE THREAD

**Issue by zeroClassSOLDIER** [spsprinkles/event-registration#1]

**Comment by zeroClassSOLDIER** [comment]
@gudatta After the page loads: Expected Result is active events should be showing. Actual Results are no data is loaded.
![DefaultStartPage](https://user-images.githubusercontent.com/84786032/142448476-800850c2-22ab-48fa-819c-e4b31c70a32c.png)

And then after toggling on the filter to show inactive the events finally show.
![AfterFilter](https://user-images.githubusercontent.com/84786032/142448853-e451b620-ce7c-48b0-831e-d71a29c5ee90.png)

I think the active events are being treated as inactive and the active events are treated as inactive. 
In other words the filter just has inactive and active swapped around.
